### PR TITLE
[Linux]Add download support

### PR DESCRIPTION
**Pull request by XiaosongWei**
Use desktop's default downloand manner to download the item

### CODE REVIEW

**Comment by XiaosongWei** [comment]
@rakuco @pozdnyakov @kenchris @seanlong @PeterWangIntel please take a look, thanks!

File: runtime/browser/runtime_download_manager_delegate.cc [hunk 1]
```diff
@@ -51,6 +55,14 @@ bool RuntimeDownloadManagerDelegate::DetermineDownloadTarget(
     content::DownloadItem* download,
     const content::DownloadTargetCallback& callback) {
   DCHECK(BrowserThread::CurrentlyOn(BrowserThread::UI));
+#if defined(OS_LINUX) && !defined(OS_TIZEN)
+  // Use platform's default download manner to download the item
```
Comment (seanlong) [inline runtime/browser/runtime_download_manager_delegate.cc]: "Use the platform's default application to handle the download URL"?

**Comment by seanlong** [comment]
This download feature is requested by Deepin, LGTM with comment nit.

**Comment by XiaosongWei** [comment]
ping @rakuco @pozdnyakov

File: runtime/browser/runtime_download_manager_delegate.cc [hunk 2]
```diff
@@ -51,6 +55,14 @@ bool RuntimeDownloadManagerDelegate::DetermineDownloadTarget(
     content::DownloadItem* download,
     const content::DownloadTargetCallback& callback) {
   DCHECK(BrowserThread::CurrentlyOn(BrowserThread::UI));
+#if defined(OS_LINUX) && !defined(OS_TIZEN)
+  // Use the platform's default application to handle the download URL
+  if (default_download_path_.empty() &&
+      download->GetForcedFilePath().empty()) {
```
Comment (pozdnyakov) [inline runtime/browser/runtime_download_manager_delegate.cc]: where this conditions come from?

Comment (XiaosongWei) [inline runtime/browser/runtime_download_manager_delegate.cc]: The conditions are used to ensure the download is offloaded to platform's default URL download application when these is no download path preset. default_download_path_ is the path set by xwalk_browsertest only.

Comment (pozdnyakov) [inline runtime/browser/runtime_download_manager_delegate.cc]: this needs a comment. lgtm otherwise

Comment (XiaosongWei) [inline runtime/browser/runtime_download_manager_delegate.cc]: Done. @pozdnyakov

**Pull request merged**